tracer/build/supported_versions.json

```diff
@@ -0,0 +1,1071 @@
+        "minVersionTestedInclusive": "0.61.0",
+        "maxVersionSupportedInclusive": "2.2.7",
+        "maxVersionAvailableInclusive": "2.2.7",
+        "maxVersionTestedInclusive": "2.2.7"
+      }
+    ]
+  },
+  {
+    "integrationName": "NLog",
+    "assemblyName": "NLog",
+    "minAssemblyVersionInclusive": "1.0.0",
+    "maxAssemblyVersionInclusive": "5.65535.65535",
+    "packages": [
+      {
+        "name": "NLog",
+        "minVersionAvailableInclusive": "1.0.0.505",
+        "minVersionSupportedInclusive": "1.0.0.505",
+        "minVersionTestedInclusive": "1.0.0.505",
+        "maxVersionSupportedInclusive": "5.2.5",
+        "maxVersionAvailableInclusive": "5.2.5",
+        "maxVersionTestedInclusive": "5.2.5"
+      }
+    ]
+  },
+  {
+    "integrationName": "Npgsql",
+    "assemblyName": "Npgsql",
+    "minAssemblyVersionInclusive": "4.0.0",
+    "maxAssemblyVersionInclusive": "7.65535.65535",
+    "packages": [
+      {
+        "name": "Npgsql",
+        "minVersionAvailableInclusive": "2.0.11",
+        "minVersionSupportedInclusive": "4.0.0",
+        "minVersionTestedInclusive": "4.1.12",
+        "maxVersionSupportedInclusive": "7.0.6",
+        "maxVersionAvailableInclusive": "7.0.6",
+        "maxVersionTestedInclusive": "7.0.6"
+      }
+    ]
+  },
+  {
+    "integrationName": "NUnit",
+    "assemblyName": "nunit.framework",
+    "minAssemblyVersionInclusive": "3.0.0",
+    "maxAssemblyVersionInclusive": "3.65535.65535",
+    "packages": [
+      {
+        "name": "NUnit",
+        "minVersionAvailableInclusive": "2.5.7.10213",
+        "minVersionSupportedInclusive": "3.0.0",
+        "minVersionTestedInclusive": "3.6.1",
+        "maxVersionSupportedInclusive": "3.13.3",
+        "maxVersionAvailableInclusive": "3.13.3",
+        "maxVersionTestedInclusive": "3.13.3"
+      }
+    ]
+  },
+  {
+    "integrationName": "OpenTelemetry",
+    "assemblyName": "OpenTelemetry",
+    "minAssemblyVersionInclusive": "1.0.0",
+    "maxAssemblyVersionInclusive": "1.0.0",
+    "packages": [
+      {
+        "name": "OpenTelemetry",
+        "minVersionAvailableInclusive": "1.0.0",
+        "minVersionSupportedInclusive": "1.0.0",
+        "minVersionTestedInclusive": null,
+        "maxVersionSupportedInclusive": "1.6.0",
+        "maxVersionAvailableInclusive": "1.6.0",
+        "maxVersionTestedInclusive": null
+      }
+    ]
+  },
+  {
+    "integrationName": "OpenTelemetry",
+    "assemblyName": "OpenTelemetry.Api",
+    "minAssemblyVersionInclusive": "1.0.0",
+    "maxAssemblyVersionInclusive": "1.0.0",
+    "packages": [
+      {
+        "name": "OpenTelemetry.Api",
+        "minVersionAvailableInclusive": "1.0.1",
+        "minVersionSupportedInclusive": "1.0.1",
+        "minVersionTestedInclusive": "1.0.1",
+        "maxVersionSupportedInclusive": "1.6.0",
+        "maxVersionAvailableInclusive": "1.6.0",
+        "maxVersionTestedInclusive": "1.6.0"
+      }
+    ]
+  },
```

Comment:
Do you know if this would cause any issues with the framework team when parsing this file? Specifically, the fact that one "package" isn't tested but in reality it is just underneath the different `assemblyName`

Reply:
🤔 Interesting... that's a good question. It's all a bit weird tbh, because of the way the packages work. I think _technically_ we kind of _do_ test this package, right? Because we 
reference `OpenTelemetry` which transitively references `OpenTelemetry.Api` (if you dig far enough through the dependencies!)

Which means this is all a bit squiff - we've marked the OpenTelemetry integration above as untested too, whereas we _do_ test that 😂 

There's also _technically_ some lies in there... we don't run the CosmosDb tests in CI (though we can, theoretically), yet we _say_ we test them...

Maybe the best option is to "manually" hack fixes for these 🤔

Reply:
I'm going to merge this for now so we can start using the Include/Exclude ASAP and look at improving the output of these in a subsequent PR